### ISSUE TITLE
[A11Y] OMP Reader View - Book of the month carroussel component uses links numbered illogically

### ISSUE THREAD

**Issue by israelcefrin** [pkp/pkp-lib#12666]
## Issue Description
On the Main Page, the "Book of the month" links for articles or books within a carrousel are numbered when accessed by screen readers, but the numbering does not follow any particular or logical order. This arbitrary sequence causes confusion for screen reader users and fails to convey the actual structure or relationship of the books presented on the page.

## How to reproduce / Current behavior
1. Navigate to the OMP Front End Main page.
2. Utilize a screen reader (such as JAWS, NVDA, or VoiceOver) to read through the list of books/articles.
3. Observe that the links are announced with numbers that are out of sequence and do not align with the visual layout or a logical sorting order.

## What application are you using?
OMP version 3.5.0

## Stack used 
- Device: Desktop
- OS: Windows
- Browser: Chrome
- Version: Latest

## Testing method
- Method: Manual; Screen Reader (JAWS or NVDA) / Keyboard

## Action
The carrousel component might be a deprecated element. Investigate the underlying HTML or ARIA attributes generating these numbers. Ensure that the programmatic reading order matches the visual order and that any numbering applied to lists or links follows a sequential, meaningful structure. The better approach is to remove it to prevent user confusion and replacing if with a static card element.

### Screenshot

<img width="1058" height="357" alt="Book of the month component" src="https://github.com/user-attachments/assets/21040e61-734e-47d9-a76c-cc6631673303" />

### Current code
```
<ol class="swiper-wrapper" style="height: 110px;" id="swiper-wrapper-d310ea78511c9f83c" aria-live="polite">
                            <li class="swiper-slide swiper-slide-active" style="width: 860px;" role="group" aria-label="1 / 1">
                                        <div class="swiper-slide-content">
```

### WCAG Success Criteria
1.3.2 Meaningful Sequence (Level A)
2.4.3 Focus Order (Level A)
